libs/features/mas/web-components/test/merch-card.ccd-slice.test.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Merch Card CCD SliceWeb Component demo page</title>
+    <script>
+        window.process = {
+            env: {},
+        };
+    </script>
+    <style>
+        main {
+            display: flex;
+            align-items: center;
+            gap: 20px;
+            flex-direction: column;
+            padding-top: 80px;
+        }
+
+        main>div {
+            display: initial;
+        }
+    </style>
+</head>
+
+<body>
+    <script type="module" src="./merch-card.ccd-slice.test.html.js"></script>
+    <main>
+        <div class="one-merch-card ccd-slice">
+          <merch-card class="" tabindex="0" variant="ccd-slice" filters="" types="">
+            <sp-status-light size="l" variant="info" dir="ltr"></sp-status-light>
+            <merch-icon
+              slot="icons"
+              src="https://www.adobe.com/content/dam/shared/images/product-icons/svg/photoshop.svg"
+              alt=""
+              href=""
+              size="l"
+              >undefined</merch-icon
+            >
+            <merch-icon
+              slot="icons"
+              src="https://www.adobe.com/content/dam/shared/images/product-icons/svg/lightroom.svg"
+              alt=""
+              href=""
+              size="l"
+              >undefined</merch-icon
+            >
+            <p slot="body-s">
+              <strong>Creative Cloud Photography plan.<br>
+              Starting at US$19.99/mo
+              </strong>
+            </p>
+            <div slot="footer">
+              <a
+                  class="con-button blue"
+                  is="checkout-link"
+                  data-checkout-workflow="UCv3"
+                  data-checkout-workflow-step="segmentation"
+                  data-promotion-code="CCI2AAUSQ22440CH"
+                  data-quantity="1"
+                  data-wcs-osi="r_JXAnlFI7xD6FxWKl2ODvZriLYBoSL701Kd1hRyhe8"
+                  data-extra-options="{}"
+                  ><span>Buy now</span></a
+                >
+            </div>
+            <div slot="image">
+              <img src="./img/creative-cloud.png" alt="">
+            </div>
+            </merch-card
+          >
+</div>
+<div class="one-merch-card ccd-slice-wide">
```

Comment:
I think we don't need this div neither since we do not add responsive grid styles for CCD cards.